File: AreaTriangulo.cpp
#include <iostream>
int main () {
     //Definir variables y otros  
     printf("Ejercicio 01:Area del Triangulo\n");
     int b=0, h=0, area=0;
     //Datos de entrada
     printf("Ingrese Base:");
     scanf("%i", &b);
     printf("Ingrese Altura:");
     scanf("%i", &h);
     //Proceso
     area=(b*h)/2;
    //Datos de Salida
    printf("Area del Triangulo es:%i %s", area, "\n");
    return 0;
}

File: AreaTriangulo.go
package main
import "fmt"
func main() {
 //Declaracion de variables y otros
 fmt.Println("Ejercicio 01: Area Triangulo")
 var b int
 var h int
 var area float64
 //Datos de lectura
 fmt.Println("Ingrese Base:")
 fmt.Scanln(&b)
 fmt.Println("Ingrese Altura:")
 fmt.Scanln(&h)
 //Proceso
 area=float64((b*h)/2)
 //Datos de salida
 fmt.Println("El area es: ", area)
} 
